### PR TITLE
New version: py-pkgconfig 1.5.5

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pkgconfig/package.py [hunk 1]
```diff
@@ -18,5 +18,5 @@ class PyPkgconfig(PythonPackage):
 
     depends_on('python@2.6:2.8,3.3:', type=('build', 'run'))
     depends_on('python@2.7:2.8,3.3:', when='@1.5:', type=('build', 'run'))
-    depends_on('py-setuptools', when='@:1.4', type=('build', 'run'))
+    depends_on('py-setuptools', when='@:1.4,1.5.1:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pkgconfig/package.py]: Where do you see this? I think it needs poetry instead according to the `pyproject.toml`

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-pkgconfig/package.py]: The pypi tarball comes with setuptools script

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pkgconfig/package.py]: Yes, but `setup.py` doesn't import `setuptools`.

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-pkgconfig/package.py]: Fixed. `setup.py` in 1.5.5 does import setuptools

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**